### PR TITLE
Make BRANCH AMENITIES fieldset collapsible

### PR DESCRIPTION
**Pull request by Fant0m771**
Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature, please create PR against 8.x-1.xx-dev branch.

If this is a bug fix - use 8.x-1.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

issue https://github.com/ymcatwincities/openy/issues/1133

## Steps for review

- [ ] Go to admin area and add new content "Branch" type
- [ ] On adding page you should see "AMENITIES" collapsed fieldset

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_form_display.node.branch.default.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ dependencies:
     - node.type.branch
   module:
     - address
+    - content_moderation
```
Comment (ddrozdik) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_form_display.node.branch.default.yml:23]: Why content moderation added? there is no request to add it in the original issue.

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_form_display.node.branch.default.yml [hunk 2]
```diff
@@ -117,6 +118,19 @@ third_party_settings:
...
+      children:
+        - field_location_amenities
+      parent_name: ''
+      weight: 8
+      format_type: tab
+      format_settings:
+        id: ''
+        classes: ''
+        formatter: open
+        description: ''
```
Comment (ddrozdik) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_form_display.node.branch.default.yml:131]: Please provide a description, that will provide information about Amenities to the end user.

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_form_display.node.branch.default.yml [hunk 3]
```diff
@@ -117,6 +118,19 @@ third_party_settings:
...
+    group_amenities:
+      children:
+        - field_location_amenities
+      parent_name: ''
+      weight: 8
+      format_type: tab
+      format_settings:
+        id: ''
+        classes: ''
+        formatter: open
```
Comment (ddrozdik) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_form_display.node.branch.default.yml:130]: should be collapsed by default

**Review by ddrozdik — CHANGES_REQUESTED** [review]
Upgrade path is missing.

**Comment by ddrozdik** [comment]
@Fant0m771 will you be able to finish this PR?

**Review by podarok — CHANGES_REQUESTED** [review]
@Fant0m771 please fix review comments

**Comment by ddrozdik** [comment]
@Fant0m771 any plans to finish work on this PR?

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by AndreyMaximov** [comment]
@podarok this one can be closed as it was finalized in #1925

**Pull request closed without merging**